.github/workflows/tests.yml

```diff
@@ -179,6 +179,10 @@ jobs:
       run: |
         pip install --progress-bar off .
 
+    - name: Uninstall PlasmaPy with pip
+      run: |
+        pip uninstall plasmapy
+
```

Comment:
To make sure that the environment is fresh before following up with the next step, which is to try an editable installation.